docs/faq.md

```diff
@@ -0,0 +1,23 @@
+# Frequently asked questions
+
+## Can I automatically tell JupyterHub what kind of server to start (node size, profile name, etc) as part of my nbgitpuller link?
+
+You can use Kubespawner's profile_list or ProfileSpawner to allow your
+end users to choose the resources (memory, cpu, GPUs, etc) they want before
+starting their server. Wouldn't it be nice if this information could be
+embedded in the nbgitpuller link, so this (often confusing) choice is made
+for your students?
+
+While it would indeed be very nice, this is not currently possible for two
```

Comment:
I thought this was inaccurate, but wasn't sure - that promoted the investigation.

I'd like this to be either closed pending work outlined in https://github.com/2i2c-org/infrastructure/issues/3096, or updated so it doesnt claim its not possible. The claim could instead be that its currently undocumented how to make a launch link specifying user options practically, and that currently with jh 4.0.2 at least, such links would not restart or error etc if a server was already running with either the same or different server options, but instead quickly proceed to the user server and the git pulling.

Reply:
@consideRatio I've updated it to say 'not easy', and then also stated that this would have to be configured at the JupyterHub level, rather than at nbgitpuller level. What do you think now?